### PR TITLE
Add a few links to markdown and rst guides in the contributing file

### PR DESCRIPTION
**Pull request by KirstieJane**
Closes # NA (I didn't open an issue 🤦‍)

Changes proposed in this pull request:

- Add a section about writing in markdown to the contributing guidelines
- Add a section about writing in rst to the contributing guidelines

I need to figure out how to credit @jdkent for his rst text that I took from this PR: https://github.com/HBClab/NiBetaSeries/pull/77

### CODE REVIEW

**Comment by KirstieJane** [comment]
BAH - don't merge this - I have no idea what I've done in trying to be nice to @jdkent!

I'm boarding a plane - I'll try to git fix this when I land 🤞

**Comment by emdupre** [comment]
Just ping me when this is ready for a review !

File: CONTRIBUTING.md
Comment (emdupre) [inline CONTRIBUTING.md:37]: I'm going to update this and restructure a bit of the text before merging, per our discussion on Slack ! thanks so much for adding this :sparkles: :hugs:

**Review by emdupre — APPROVED** [review]

**Pull request merged**
